qcdb/moptions/options.py

```diff
@@ -88,7 +88,7 @@ def prepare_options_for_cfour(options):
 
     for key, ropt in sorted(options.scroll['CFOUR'].items()):
         #if not ropt.is_default():
-        if ropt.disputed():
+        if ropt.disputed() and ropt.value is not None:
```

Comment:
It looks like `qcdb.driver.pe.clean_nu_options()` resets all options, which isn't exactly what was needed here. I was trying to reset a single option.

The situation that required this: I was performing alternating unfrozen and frozen core calculations in CFOUR. Freezing the core works great (something like `qcdb.set_options({'cfour_dropmo': [1,2]})`). Un-freezing, however, didn't seem to work. Setting the frozen core to an empty list (`qcdb.set_options({'cfour_dropmo': []})`) crashes CFOUR. This seemed like the next best option.

Reply:
got it. `None` isn't quite the right way to solve this because qcdb options maintain a history, so you just want to clear away the user setting rather than assert something else (even if None). so i'll undo this but seek a replacement.